xml/System/AppDomain.xml

```diff
@@ -55,6 +55,9 @@
   
 -   If an assembly is loaded into the default application domain, it cannot be unloaded from memory while the process is running. However, if you open a second application domain to load and execute the assembly, the assembly is unloaded when that application domain is unloaded. Use this technique to minimize the working set of long-running processes that occasionally use large DLLs.  
   
+ > [!NOTE]
+ >  On .NET Core, the <xref:System.AppDomain> implementation is limited by design and does not provide isolation, unloading, or securtity boundaries.  For .NET Core there is exactly one <xref:System.AppDomain>.   Isolation and unloading areß provided through <xref:System.Runtime.Loader.AssemblyLoadContext>.  Security boundaries should be provided by process boundaries and appropriate remoting techniques.
```

Comment:
- securtity --> security
- comma after "For .NET Core"
- only one space between period and "Isolation"
- areB --> are